From: Yaroslav Halchenko <debian@onerussian.com>
Date: Thu, 14 May 2020 12:54:06 +0200
Subject: Disable google analytics referrals in the -doc

  if we were to provide some information from just reading  documentation we
  had to disclose and seek approval from the user before doing that.

  Thanks Ambrose Andrews for the analysis and report

Origin: Debian
Bug: 730259
Bug-Debian: http://bugs.debian.org/730259
Bug-Ubuntu: https://launchpad.net/bugs/
Last-Update: 2013-11-24
---
 doc/conf.py                        | 2 +-
 doc/themes/scikit-learn/theme.conf | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

Index: scikit-learn/doc/conf.py
===================================================================
--- scikit-learn.orig/doc/conf.py	2021-11-27 09:51:28.684414912 +0100
+++ scikit-learn/doc/conf.py	2021-11-27 09:51:58.950282571 +0100
@@ -163,7 +163,7 @@
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
-html_theme_options = {"google_analytics": True, "mathjax_path": mathjax_path}
+html_theme_options = {"google_analytics": False, "mathjax_path": mathjax_path}
 
 # Add any paths that contain custom themes here, relative to this directory.
 html_theme_path = ["themes"]
